src/lib/slider/slider.spec.ts

```diff
@@ -311,7 +316,33 @@ describe('MdSlider', () => {
       // The closest snap is at the halfway point on the slider.
       expect(thumbDimensions.left).toBe(sliderDimensions.left + sliderDimensions.width * 0.5);
       expect(trackFillDimensions.width).toBe(thumbPosition);
+    });
+
+    it('should adjust thumb and ticks when min changes', () => {
+      testComponent.min = -2;
+      fixture.detectChanges();
 
+      let trackFillDimensions = trackFillElement.getBoundingClientRect();
+      let tickContainerDimensions = tickContainerElement.getBoundingClientRect();
+
+      expect(trackFillDimensions.width).toBe(sliderDimensions.width * 6 / 8);
+      expect(tickContainerDimensions.width)
+          .toBe(sliderDimensions.width - sliderDimensions.width * 6 / 8);
```

Comment:
I believe we also need to check that the linear gradient background for the ticks has been updated (since the tick interval will have changed).